routes/merged/project/location/load_balancer.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+class Clover
+  branch = lambda do |r|
+    r.get api? do
+      load_balancer_list
+    end
+
+    r.on NAME_OR_UBID do |lb_name, lb_id|
+      filter = if lb_name
+        {Sequel[:load_balancer][:name] => lb_name}
+      else
+        {Sequel[:load_balancer][:id] => UBID.to_uuid(lb_id)}
+      end
+
+      filter[:private_subnet_id] = @project.private_subnets_dataset.where(location: @location).select(Sequel[:private_subnet][:id])
+      lb = LoadBalancer.first(filter)
+
+      unless lb
+        response.status = request.delete? ? 204 : 404
+        request.halt
+      end
+
+      r.post %w[attach-vm detach-vm] do |action|
+        Authorization.authorize(current_account.id, "LoadBalancer:edit", lb.id)
+        required_parameters = %w[vm_id]
+        request_body_params = Validation.validate_request_body(json_params, required_parameters)
+        vm = Vm.from_ubid(request_body_params["vm_id"])
+
+        unless vm
+          response.status = 404
+          if api?
+            r.halt
+          else
+            flash["error"] = "VM not found"
+            r.redirect "#{@project.path}#{lb.path}"
+          end
+        end
+
+        Authorization.authorize(current_account.id, "Vm:view", vm.id)
+
+        if action == "attach-vm"
+          if vm.load_balancer
+            flash["error"] = "VM is already attached to a load balancer"
+            response.status = 400
+            r.redirect "#{@project.path}#{lb.path}"
+          end
+          lb.add_vm(vm)
+          actioned = "attached"
```

Comment:
This should probably be "attached to", and detach should use "detached from", and the flash notice should remove the " from".  This is because the flash notice should say "attached to the load balancer" and not "attached from the load balancer".

Reply:
Done